modules/admin_manual/pages/configuration/files/encryption/encryption_configuration.adoc

```diff
@@ -14,19 +14,22 @@ configuration/files/encryption/moving-key-locations.adoc, \
 configuration/files/encryption/sharing-encrypted-files.adoc, \
 configuration/files/encryption_configuration.adoc
 
+:description: The primary purpose of the ownCloud server-side encryption is to protect users’ files when they’re located on remote storage sites such as Dropbox and Google Drive, smoothly and seamlessly from within ownCloud.
+
 == Introduction
 
-The primary purpose of the ownCloud server-side encryption is to protect users’ files when they’re located on remote storage sites, such as Dropbox and Google Drive, and to do it smoothly and seamlessly from within ownCloud.
+{description}
+
+== General Notes
 
-Since ownCloud 9.0, server-side encryption for local and remote storage can operate independently. This allows you to encrypt a remote storage _without_ also having to encrypt your home storage on your ownCloud server.
+Since ownCloud 9.0, server-side encryption for local and remote storage can operate independently. This allows to encrypt a remote storage independent of the home storage on the ownCloud server.
```

Comment:
```suggestion
Since ownCloud 9.0, server-side encryption for local and remote storage can operate independently. This allows encrypting a remote storage independent of the home storage on the ownCloud server.
```